test/extended/cli/explain.go

```diff
@@ -188,6 +188,14 @@ var (
 		{Group: "metal3.io", Version: "v1alpha1", Resource: "provisionings"},
 	}
 
+	nodeTuningTypes = []schema.GroupVersionResource{
+		{Group: "tuned.openshift.io", Version: "v1", Resource: "profiles"},
+		{Group: "tuned.openshift.io", Version: "v1", Resource: "tuneds"},
```

Comment:
I think you want to drop these two entries from `baseCRDTypes` up around line 152.  Not clear to me why `performanceprofiles` aren't mentioned in the `master` tip, but :shrug:, I guess you're adding them now.